os_stub/spdm_device_secret_lib_sample/cert.c

```diff
@@ -496,6 +496,152 @@ bool libspdm_read_responder_public_certificate_chain(
     return true;
 }
 
+bool libspdm_read_responder_public_certificate_chain_alias_cert(
```

Comment:
This name is confusing.

Is this full alias cert chain till leaf? 
or alias cert chain till device cert CA?